README.md

```diff
@@ -34,6 +52,38 @@ It is a standalone Microservice that connects the two platforms and provides a s
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
+## Supported payment methods
+
+The PAYONE Integration Service currently supports the following payment methods:
```

Comment:
Add a link to payone doc and/or add short descriptions of the payment methods. If there is further information documented e.g. Klarna activation - also link it.

Reply:
fab8376eb